packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -1106,7 +1106,7 @@ class TimeInput extends React.Component {
             {...inputAttributes}
             {...minuteAttributes}
             refCallback={(inputRef) => { this.minuteInput = inputRef; }}
-            label={intl.formatMessage({ id: 'Terra.timeInput.minutes' })}
+            label={intl.formatMessage({ id: 'Terra.timeInput.minutes' }, { a11yLabel: this.a11yLabel })}
```

Comment:
why do we need to send this as object value ..??

Reply:
In this case i have followed the previous code standard as it is being used for Hours field